src/components/NodeStatus/NodeStatus.js

```diff
@@ -25,10 +24,14 @@ const UnderMaintenanceStatus = ({ node, maintenance, TimestampComponent }) => {
       <div>
         Started: <TimestampComponent simple timestamp={created} />
       </div>
-    </Popover>
+    </React.Fragment>
   );
 
-  return <OverlayStatus icon="off" overlay={overlay} text="Under maintenance" />;
+  return (
+    <OverlayStatus icon="off" header="Under maintenance">
+      {overlay}
```

Comment:
i'd just move the whole overlay content here

Reply:
Nice :+1: